test/actions/is_present_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule IsPresentPage do
```

Comment:
Since this is a query specifically for presence, I think it would be worth adding a test for an element that has a style of `display: none;` to confirm it behaves as we expect